package.json

```diff
@@ -45,11 +45,13 @@
     "indexer-codegen": "graphql-codegen --config ./src/types/codegen.yaml && pnpm fmt",
     "doc": "scripts/generateDocs.sh",
     "check-version": "scripts/checkVersion.sh",
-    "update-version": "scripts/updateVersion.sh && pnpm doc"
+    "update-version": "scripts/updateVersion.sh && pnpm doc",
+    "spec": "pnpm build && pnpm _spec",
+    "_spec": "./node_modules/.bin/cucumber-js -p default"
```

Comment:
why are we using the bin file?

Reply:
This is a prototype for the moment, I don't expect to merge it in necessarily.

There is this weird conflict, where I needed to use ESM, but it kept giving me conflicts about loading CJS, rather than ESM.

Would love to sit down with someone who's more familiar with it than me and I'm sure we could knock it out.

Reply:
Figured it out, need to build prior to running.  Should be fixed now.